go/pkg/client/client.go

```diff
@@ -267,6 +268,10 @@ type DialParams struct {
 
 	// MaxConcurrentStreams specifies the maximum number of concurrent stream RPCs on a single connection.
 	MaxConcurrentStreams uint32
+
+	// TLSClientAuthCert/TLSClientAuthKey sets the cert/key pair for using mTLS auth to connect to the RBE service.
```

Comment:
To pass golint, the comments on an exported variable need to begin with the variable name (so you need a separate comment per line).
As an aside, all of these variables should not be exported, and I will clean this up separately...